File: code/scratchpad/hello-rust/hello.rs
pub fn add(num1: u32, num2: u32) -> u32 {
    return num1 + num2;
}

fn main() {
    // calling a function
    let num1: u32 = 10;
    let num2: u32 = 20;
    println!("Sum of {} and {} is {}", num1, num2, add(num1, num2));

    // looping
    let mut result: u32 = 0;
    for item in 0..100 {
        if item > 10 {
            result += item * 2;
        } else {
            result += item * 11;
        }
    }
    println!("Value at the end of the loop is {}", result);

}

File: code/scratchpad/hello-c/hello.c
#include <stdio.h>

int add(int num1, int num2)
{
    return num1 + num2;
}

int main()
{
    // Add two numbers
    int num1 = 2341;
    int num2 = 1978;
    int sum = add(num1, num2);
    printf("Sum of %d and %d is %d.\n", num1, num2, sum);

    // Loop through some numbers
    int result = 0;
    for (int i = 0; i < 100; ++i)
    {
        if (i > 10)
        {
            result += i * 2;
        }
        else
        {
            result += i * 11;
        }
    }
    printf("%d\n", result);
    return 0;
}
